tests/OSPSuite.Core.IntegrationTests/ContextForIntegration.cs

```diff
@@ -105,6 +105,7 @@ public void RegisterComponents(CastleWindsorContainer container)
          IoC.RegisterImplementationOf(IoC.Container);
 
          container.WindsorContainer.AddFacility<TypedFactoryFacility>();
+         container.Register<IEntityValidatorFactory, EntityValidatorFactory>();
```

Comment:
Why are we registering this by hand?

Reply:
this is only in tests right? I didn't think the scan was used here.

Reply:
hum weird. I don't know why we would register everything by hand in our integrationstests. Kinds of defeat the purpose of integration somehow lol. Forget it then :)